resources/zib/zib-MedicationAgreement.xml

```diff
@@ -0,0 +1,269 @@
+      <type>
+        <code value="Extension" />
+        <profile value="http://nictiz.nl/fhir/StructureDefinition/ext-PeriodOfUse" />
+      </type>
+    </element>
+    <element id="MedicationRequest.extension:periodOfUse.value[x]">
+      <path value="MedicationRequest.extension.value[x]" />
+      <definition value="**Start date**: This is the time at which the agreement was to take effect (or took effect or will take effect). This is the time at which the instructions for use in this agreement start. In the case of an agreement to discontinue use, this is the start date of the original medication agreement. The end date indicates from when the medication is to be discontinued.&#xD;&#xA;&#xD;&#xA;**End date**: The time at which the period of use ends (or ended or will end). In the case of an agreement to discontinue use, this is the time at which the medication is to be discontinued. To avoid confusion between 'to' and 'up to', the submission of time is always mandatory for the end date.&#xD;&#xA;&#xD;&#xA;With medication for an indefinite period only a start date is indicated." />
+    </element>
+    <element id="MedicationRequest.extension:periodOfUseDuration">
+      <path value="MedicationRequest.extension" />
+      <sliceName value="periodOfUseDuration" />
+      <type>
+        <code value="Extension" />
+        <profile value="http://nictiz.nl/fhir/StructureDefinition/ext-PeriodOfUse-Duration" />
+      </type>
+    </element>
+    <element id="MedicationRequest.extension:periodOfUseDuration.value[x]">
+      <path value="MedicationRequest.extension.value[x]" />
+      <definition value="**Duration**: The intended duration of use. E.g. 5 days or 8 weeks. It is not allowed to indicate the duration in months, because different months have a variable duration in days." />
+    </element>
+    <element id="MedicationRequest.modifierExtension">
+      <path value="MedicationRequest.modifierExtension" />
+      <slicing>
+        <discriminator>
+          <type value="value" />
+          <path value="url" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="MedicationRequest.modifierExtension:stopType">
+      <path value="MedicationRequest.modifierExtension" />
+      <sliceName value="stopType" />
+      <max value="1" />
+      <type>
+        <code value="Extension" />
+        <profile value="http://nictiz.nl/fhir/StructureDefinition/ext-StopType" />
+      </type>
+    </element>
+    <element id="MedicationRequest.modifierExtension:repeatPeriodCyclicalSchedule">
+      <path value="MedicationRequest.modifierExtension" />
+      <sliceName value="repeatPeriodCyclicalSchedule" />
+      <max value="1" />
+      <type>
+        <code value="Extension" />
+        <profile value="http://nictiz.nl/fhir/StructureDefinition/ext-InstructionsForUse.RepeatPeriodCyclicalSchedule" />
+      </type>
+    </element>
+    <element id="MedicationRequest.category">

```

Comment:
`MedicationRequest.status` is required and has an indirect relationship with the zib concepts MedicationAgreementStopType and PeriodOfUse. I think an indirect mapping should be mapped to it.